about.html

```diff
@@ -82,7 +83,7 @@ <h2 class="text text-center">Health Exhibition and Confress logo</h2>
             trade fair in Ethiopia</p>
         </div>
         <div class="logo-b">
```

Comment:
- [ ] This section has a heading , a paragraph of text and two articles. The paragraph is missing from your design, kindly fix this issue

Your design
![image](https://user-images.githubusercontent.com/81932478/205450214-75c93b3d-f91a-44ec-919b-9aa8214d73fa.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205450226-a10c719a-8fc3-47f5-aced-589dbb18506d.png)